pkg/cmd/pr/shared/display.go

```diff
@@ -6,44 +6,46 @@ import (
 	"strings"
 
 	"github.com/cli/cli/api"
+	"github.com/cli/cli/pkg/iostreams"
 	"github.com/cli/cli/utils"
 )
 
-func StateTitleWithColor(pr api.PullRequest) string {
-	prStateColorFunc := ColorFuncForPR(pr)
+func StateTitleWithColor(cs *iostreams.ColorScheme, pr api.PullRequest) string {
+	prStateColorFunc := cs.ColorFromString(ColorForPR(pr))
+
 	if pr.State == "OPEN" && pr.IsDraft {
 		return prStateColorFunc(strings.Title(strings.ToLower("Draft")))
 	}
 	return prStateColorFunc(strings.Title(strings.ToLower(pr.State)))
 }
 
-func ColorFuncForPR(pr api.PullRequest) func(string) string {
+func ColorForPR(pr api.PullRequest) string {
 	if pr.State == "OPEN" && pr.IsDraft {
-		return utils.Gray
+		return "gray"
 	}
-	return ColorFuncForState(pr.State)
+	return ColorForState(pr.State)
 }
 
-// ColorFuncForState returns a color function for a PR/Issue state
-func ColorFuncForState(state string) func(string) string {
+// ColorForState returns a color constant for a PR/Issue state
+func ColorForState(state string) string {
 	switch state {
 	case "OPEN":
-		return utils.Green
+		return "green"
 	case "CLOSED":
-		return utils.Red
+		return "red"
 	case "MERGED":
-		return utils.Magenta
+		return "magenta"
 	default:
-		return nil
+		return ""
 	}
 }
 
-func PrintHeader(w io.Writer, s string) {
-	fmt.Fprintln(w, utils.Bold(s))
+func PrintHeader(cs *iostreams.ColorScheme, w io.Writer, s string) {
+	fmt.Fprintln(w, cs.Bold(s))
 }
 
-func PrintMessage(w io.Writer, s string) {
-	fmt.Fprintln(w, utils.Gray(s))
+func PrintMessage(cs *iostreams.ColorScheme, w io.Writer, s string) {
```

Comment:
Same as above comment.

Reply:
I went back and forth, there; I went with this pattern because I wanted those functions to be able to continue to work with whatever generic writer instead of hardcoding an output handle but since there's no actual dynamic behavior around that I suppose there is no pain in hardcoding it.